src/main/java/com/specialwarriors/conal/contribution/scheduler/ContributionScheduler.java

```diff
@@ -26,10 +28,12 @@ public void sendContribution() {
         List<NotificationAgreement> notificationAgreements = notificationAgreementQuery
             .findAllByType(NotificationType.CONTRIBUTION);
 
-        List<GithubRepo> githubRepos = notificationAgreements.stream()
-            .map(NotificationAgreement::getGithubRepo)
+        List<Long> githubRepoId = notificationAgreements.stream()
+            .map(NotificationAgreement::getGithubRepoId)
             .toList();
```

Comment:
```suggestion
        List<Long> githubRepoIds = notificationAgreements.stream()
            .map(NotificationAgreement::getGithubRepoId)
            .toList();
```
리스트 타입의 변수가 단수형으로 지어질 경우 의미가 불명확해진다고 생각합니다. 수정 부탁드립니다.

Reply:
> 리스트 타입의 변수가 단수형으로 지어질 경우 의미가 불명확해진다고 생각합니다. 수정 부탁드립니다.

넵!